test/Unit/Locator/GitRepositoryDirLocatorTest.php

```diff
@@ -69,7 +69,10 @@ public function it_can_passthrough_git_dir_path_if_file_is_not_parseable(): void
      */
     public function it_can_locate_git_dir_in_workspaces(): void
     {
-        $this->filesystem->dumpFile($this->gitDir, 'gitdir: /dev/null');
-        $this->assertEquals('/dev/null', $this->locator->locate($this->gitDir));
+        $worktreeRoot = $this->workspace.'/git_root/worktrees/git_worktree/';
```

Comment:
maybe better to name `git_root` -> `.git`
and maybe `git_worktree` to `worktree1` or `myworktree` (since I though this might have been a git folder)

Reply:
Clarified it a little